release_notes/47-release-notes.adoc

```diff
@@ -0,0 +1,348 @@
+
+With {product-title-short} 4.7, you can now see the status of a violation directly on the *Violations* page, and determine if the violation has been resolved. This enhances automation, as the links to the alerts now show the full context when they are accessed.
+
+//ROX-26335
+[id="prioritize-cves-with-the-epss-integration_{context}"]
+=== Prioritize CVEs with the EPSS integration
+
+{product-title-short} 4.7 introduces integration with the Exploit Prediction Scoring System (EPSS), a data-driven model that estimates the likelihood of a software vulnerability being exploited. 
+
+In addition to the severity and Common Vulnerability Scoring System (CVSS) score, an EPSS probability score from 0%-100% is now displayed for each detected Common Vulnerabilities and Exposures (CVE). 
+You can use the EPSS score to better prioritize the remediation of CVE vulnerabilities and strengthen your security strategy.
+
+//ROX-26330
+[id="gain-visibility-into-the-external-ips-in-the-network-graph_{context}"]
+=== Enhanced visibility into the external IPs in the network graph
+
+With {product-title-short} 4.7, you can now get a better insight into the external IPs behind external entities in the network graph. 
+This enhancement helps you identify the specific IP addresses your deployments are communicating with and gives you better insight into the external interactions and potential risks.
+
+//ROX-25007
+[id="enhanced-options-for-the-roxctl-netpol-generate-command_{context}"]
+=== Enhanced options for the roxctl netpol generate command
+
+{product-title-short} 4.7 introduces improved options for the `roxctl netpol generate` command. You can now specify DNS ports by using names as well as numbers. If you do not specify a port, port `53` is used automatically. 
+
+However, you can change this value by using the `--dnsport` flag. For example, you can now specify the port when a service has a defined name by using the `--dnsport dns`. 
+If you are using {osp} with the default DNS setting, you can change the default port by using the `--dnsport` flag, as {osp}’s DNS pod listens on port `5353`.
+
+For more information, see xref:../cli/command-reference/roxctl-netpol.adoc#roxctl-netpol-generate_roxctl-netpol[roxctl netpol generate].
+
+//ROX-24589
+//tech preview
+[id="integrate-the-vulnerability-insights-into-the-red-hat-developer-hub_{context}"]
+=== Integrate the vulnerability insights into the Red{nbsp}Hat Developer Hub
+
+:FeatureName: Integration of vulnerability insights into the Red{nbsp}Hat Developer Hub
+include::snippets/technology-preview.adoc[]
+
+In {product-title-short} 4.7, you can now bring vulnerability insights directly into the Red{nbsp}Hat Developer Hub and other platforms based on the upstream Backstage project by using the Backstage Plugin.
+You can incorporate the {product-title-short} plugin into your developer environment and review key vulnerabilities for your applications within the platform and take the necessary remediation actions.
```

Comment:
```suggestion
You can incorporate the {product-title-short} plugin into your developer environment and review key vulnerabilities for your applications within the platform. This provides information that you need before you can take the necessary remediation actions.
```
Suggesting a slight rewrite, because the way that it is written implies that you can use the plugin to remediate, which I don't think is true.